source/docs/software/advanced-controls/introduction/approaches_to_ctrl_sys_design.rst

```diff
@@ -0,0 +1,81 @@
+Approaches to Control Systems
+=============================
+
+.. note:: This article includes sections of `Controls Engineering in FRC <https://file.tavsys.net/control/controls-engineering-in-frc.pdf>`__ by Tyler Veness with permission.
+
+When desigining a control system for your robot, there are a number of different approaches to take. They range from very simple approaches, to advanced and complex ones. Each has *tradeoffs*. Some will work better than others in different situations, some require more mathematical analysis than others.
+
+Teams should prioritize picking the easiest strategy which enables success on the field. However, as you do experiemnts, keep in mind there is almost always a "next-step" to take to improve your field performance.
+
+FeedForward vs. FeedBack
+------------------------
+
+There are two broad categories of strategies in control system design:
+
+**Feed-Forward** refers to the class of algorithms which incorporate knowledge of how the mechanism under control is *expected* to operate. Using this "model" of operation, the control input is chosen to make the mechanism get close to where it should be.
+
+**Feed-Back** refers to the class of algorithms which use sensors to *measure* what a mechanism is doing, and issue corrective commands to move a mechnaism from where it actually is, to where you want it to be.
+
+They are not mutually exclusive - the most advanced techniques will incorporate both of these concepts simultaneously.

```

Comment:
> They are not mutually exclusive - the most advanced techniques will incorporate both of these concepts simultaneously.

Perhaps reword this to not showcase "the most advanced" but that this is fairly standard to combine the two.

Reply:
got em